src/main/java/org/tinyradius/attribute/RadiusAttribute.java

```diff
@@ -77,8 +77,9 @@ public int getAttributeType() {
 	 *            type code, 0-255
 	 */
 	public void setAttributeType(int attributeType) {
-		if (attributeType < 0 || attributeType > 255)
-			throw new IllegalArgumentException("attribute type invalid: " + attributeType);
+                // Vendor specific attribute has values > 255 and < 1
+		/*if (attributeType < 0 || attributeType > 255)
```

Comment:
I'd prefer you remove the commented out code, not sure what is the purpose to leave it as commented out.